packages/design/src/Label/Label.jsx

```diff
@@ -59,15 +59,16 @@ const kind = ({ kind, theme }) => {
 
 const Label = styled.div`
```

Comment:
we can probably achieve the ellipsis affect here, all with css (this block of code is old, we can change it, for example not sure why we are even using flex here, `inline-block` probably makes more sense